receiver.go

```diff
@@ -360,7 +360,8 @@ func (r *Receiver) countUnsettled() int {
 func newReceiver(source string, session *Session, opts *ReceiverOptions) (*Receiver, error) {
 	l := newLink(session, encoding.RoleReceiver)
 	l.source = &frames.Source{Address: source}
-	l.target = new(frames.Target)
+	target := new(frames.Target)
```

Comment:
Was this broken out for a reason?  It's harmless but adds noise to the diff.

Reply:
Yes, it's so we can assign fields to `target` without having to cast/typeassert. There's a bunch below.

Reply:
I thought that was only for senders? Maybe I missed it then.